christfellowshipchurch/web-app-v2
# Wistia Player

### About
This PR introduces an embedded Wistia Player. While still supporting traditional video links in our `Video.js` component, we will also be supporting the Wistia Player when passing a `wisitiaId` to the component.

### Test Instructions
* you can test any article or message with a video and see that it is working as is. We currently do not have any example videos using the Wistia player at the moment. The player would like the image below when in use:

### Screenshots
![image](https://user-images.githubusercontent.com/46049974/232818221-7b965a92-b717-4f16-a86e-307bb0d56eff.png)

### Related Tickets
[CFDP-2514](https://christfellowshipchurch.atlassian.net/browse/CFDP-2514?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ)

[CFDP-2514]: https://christfellowshipchurch.atlassian.net/browse/CFDP-2514?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
